caps/types.go

```diff
@@ -32,6 +32,9 @@ const (
 	// the capability concept before we get to load capability interfaces
 	// from YAML.
 	FileType Type = "file"
+	// testType is only meant for testing. It is not useful in any way except
+	// that it offers an simple capability type that will happily validate.
+	testType Type = "test"
```

Comment:
Gustavo was suggessting (in another review) to use export_test.go as a pattern to export stuff that is only exported for testing purposes. This looks like we should use it for this type too?